README.md

```diff
@@ -30,15 +30,18 @@ To run the entire release pipeline, do the following:
 
 For now you are required to specify the version being released, and only minor versions increments are supported.
 
-### Running the entire pipeline
+### Running a specific project step-by-step (recommended)
+
+You can use the commands documented in [learning team release process](https://github.com/ember-learn/handbook/blob/master/ember-releases.md) to run this tool step-by-step.
+
+If you need to run the pipeline for one of the projects, you can specify a `--project` (or `-p`) option when invoking the tool.
```

Comment:
This isn't true any more. The "projects" have been moved to the sub commands now so  this paragraph can be deleted